cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -110,29 +110,42 @@ func createRecorder(kubecli *kubernetes.Clientset) record.EventRecorder {
 	return eventBroadcaster.NewRecorder(scheme.Scheme, apiv1.EventSource{Component: "kubeturbo"})
 }
 
-func (s *VMTServer) createKubeConfig() (*restclient.Config, error) {
+func (s *VMTServer) createKubeConfigOrDie() *restclient.Config {
 	kubeConfig, err := clientcmd.BuildConfigFromFlags(s.Master, s.KubeConfig)
 	if err != nil {
-		glog.Errorf("Error getting kubeconfig:  %s", err)
-		return nil, err
+		panic(fmt.Sprintf("Error getting kubeconfig:  %s", err))
```

Comment:
why not use 
```go
glog.Fatalf()
```
https://godoc.org/github.com/golang/glog#Fatalf

Reply:
I was not sure which is better. But I saw k8s is using [panic](https://github.com/kubernetes/apimachinery/blob/dc1f89aff9a7509782bde3b68824c8043a3e58cc/pkg/api/resource/quantity.go#L136).  

I think it is better to call glog.Error() and exit. This is more grace.